lib/Alien/Base/ModuleBuild.pm

```diff
@@ -404,7 +416,7 @@ sub ACTION_alien_code {
 
   my $version;
   $version = $self->alien_check_installed_version
-    unless $Force;
+    unless $self->config_data('Force');
```

Comment:
This is a slight change in behavior in that it used to check the environment variable, which could potentially change between `perl Build.PL` and `./Build`.  This is how `Alien::Build` works though, and it is probably better.

Reply:
I noticed that change of behaviour. But I feel that it would make it strange, and that this way it was more coherent. But happy to suggest another option if you like.

Reply:
Yeah I think I agree